pytube/__main__.py

```diff
@@ -117,6 +118,14 @@ def init(self):
                 self.watch_html,
             )['args']
 
+        # load the player_response object (contains subtitle information)
+        apply_mixin(self.player_config_args, 'player_response', json.loads)
+
+        player_response = self.player_config_args['player_response']
+        if ('playabilityStatus' in player_response and
+                player_response['playabilityStatus']['status'] != 'OK'):
```

Comment:
`['playabilityStatus']['status'] != 'OK'` only for live streams?

Reply:
behaviour is now:

- `status` always `OK`
- live streams have `liveStreamability` key but their `stream` is an empty `dict`
- videos that were live streams have `liveStreamability` key, `stream` not an empty `dict`